apps/bigcommerce/schedule/TERMS_OF_SERVICE.md

```diff
@@ -0,0 +1,179 @@
+# Terms of Service
+
+**Effective Date: January 14, 2025**
```

Comment:
I assume this is fine but what's the general approach to dating things like this?